### DISCUSSION TITLE
Originals directory rename or directory as album name

### DISCUSSION THREAD

**Discussion by MafiaInc** [photoprism/photoprism#909] [Q&A] [answered]
What would happen if I rename/move directory inside "originals", which was indexed already?
I have ordered and named collection of photos in my originals directory and I want to keep the same naming convention when I upload from the web interface. e.g.:
currently: "originals/Album Name year.month/*.jpg"
when uploading:  "2011/01/*.jpg"
When I upload via the interface I would like somehow to have album name as directory. If this is not possible as configuration parameter I would be happy to rename my directories after they were uploaded. Then I guess I should index originals again. Would that take much time again and would that create extra metadata storage?

**Comment by lastzero** [comment, marked as answer]
We'll later implement uploading to specific folders. Right now, uploads will be imported which moves them automatically based on their date. Uploading to an existing folder is more complex as there may be conflicts (existing files), so we need to implement a conflict resolution dialog for this and can't just use the existing upload.

You are free to rename / move already indexed files and folders - PhotoPrism detects this when indexing. Please avoid switching names like renaming Folder A to B and Folder B to A in one go.
